pkg/detectors/github/github.go

```diff
@@ -48,13 +56,49 @@ type userRes struct {
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"ghp_", "gho_", "ghu_", "ghs_", "ghr_", "github_pat_"}
+	return []string{"ghp_", "gho_", "ghu_", "ghs_", "ghr_", "github_pat_", "github"}
```

Comment:
`github` matches means a lot of matches. Something like `github_client` would cut that down, but could miss creds. I erred on the side of not missing anything, but looking for other opinions.

Reply:
I'm with you, I think erring on the side of not missing anything makes sense. And to clarify a little bit here, the `github` keyword would be associated with oauth creds right? Would it make sense to use `detectors.PrefixRegex` for the oauth regexes?

Reply:
Good call. Switched.